app/components/Configuration.react.js

```diff
@@ -53,6 +54,7 @@ class Configuration extends Component {
       cookie.remove('db-connector-user');
       cookie.remove('plotly-auth-token');
       cookie.remove('db-connector-auth-token');
+      cookie.remove('auth-disabled');
```

Comment:
`db-connector-auth-disabled`? These types of typos are why we create constants and tests :)